languagetool-language-modules/ga/src/main/resources/org/languagetool/resource/ga/added.txt

```diff
@@ -0,0 +1,76 @@
+atáid	bí	Verb:VI:PresInd:3P:Pl:Rel:CM
```

Comment:
A comment added with `#` would be good, could refer to the English `added.txt` maybe